callibration: {

K_C: [1.45473647e+03,  0.00000000e+00, 2.09377220e+03,
      0.00000000e+00,  1.45515943e+03, 1.04210767e+03,
      0.00000000e+00,  0.00000000e+00, 1.00000000e+00],

R_Lidar_CameraC: [-0.99663729, -0.08081503,  0.01352938,
                  0.017167  , -0.04448773,  0.99886242,
                  -0.08012121,  0.99573579,  0.0457255],

t_Lidar_CameraC: [0.01210299,
                  0.32248433,
                  -0.03505703],

D_C: [ 0.01429048, -0.01582571, 0.01403004, -0.00528917],
}